modules/EmailMan/EmailMan.php

```diff
@@ -965,4 +995,54 @@ public function mark_deleted($id)
     {
         $this->db->query("DELETE FROM {$this->table_name} WHERE id=" . intval($id));
     }
+
+    /**
+     * @global array|Configurator $sugar_config;
+     * @param \Contact|\Account|\Prospect|\SugarBean $bean
+     * @return bool true === block email from being sent
+     */
+    private function shouldBlockEmail(SugarBean $bean)
+    {
+        global $sugar_config;
+
+        $optInLevel = isset($sugar_config['email_enable_confirm_opt_in']) ? $sugar_config['email_enable_confirm_opt_in'] : '';
+
+        // Find email address
+        $email_address = trim($bean->email1);
```

Comment:
Function parameter is a `SugarBean` object but class SugarBean hasn't $email1 variable in it. May it will shows up a php notice.